test/Microsoft.AspNet.Identity.Test/IdentityResultTest.cs

```diff
@@ -24,29 +24,5 @@ public void NullFailedUsesEmptyErrors()
             Assert.False(result.Succeeded);
             Assert.Equal(0, result.Errors.Count());
         }
-
```

Comment:
ah no we aren't. I have fixed it locally and will be in the final PR. We now check the log messages as well as scopes.